crowbar_framework/app/assets/javascripts/barclamps/neutron/application.js

```diff
@@ -215,9 +215,10 @@
   };
 
   CiscoPorts.prototype.visualSwitches = function() {
-    return !$.isEmptyObject(this.retrieveSwitches())
-      && $('#networking_plugin').val() == 'cisco'
-      && $('#networking_mode').val() == 'vlan';
+      return !$.isEmptyObject(this.retrieveSwitches())
```

Comment:
> Wrong indent, remove 2 spaces.

All js indent done in https://github.com/toabctl/barclamp-neutron/commit/bb6f0757b4155c9fb84e87254977e84028a66cbc